packages/contracts-bedrock/src/L1/OPContractsManagerStandardValidator.sol

```diff
@@ -22,22 +22,21 @@ import { IETHLockbox } from "interfaces/L1/IETHLockbox.sol";
 import { IPermissionedDisputeGame } from "interfaces/dispute/IPermissionedDisputeGame.sol";
 import { IProxyAdmin } from "interfaces/universal/IProxyAdmin.sol";
 import { IDelayedWETH } from "interfaces/dispute/IDelayedWETH.sol";
+import { ISemver } from "interfaces/universal/ISemver.sol";
 import { IResourceMetering } from "interfaces/L1/IResourceMetering.sol";
 import { IOptimismPortal2 } from "interfaces/L1/IOptimismPortal2.sol";
 import { IPreimageOracle } from "interfaces/cannon/IPreimageOracle.sol";
 import { IMIPS64 } from "interfaces/cannon/IMIPS64.sol";
-import { ISemver } from "interfaces/universal/ISemver.sol";
-import { IProxyAdminOwnedBase } from "interfaces/L1/IProxyAdminOwnedBase.sol";
 
-/// @title OPContractsManagerStandardValidator
+/// @title StandardValidator
 /// @notice This contract is used to validate the configuration of the L1 contracts of an OP Stack chain.
 /// It is a stateless contract that can be used to ensure that the L1 contracts are configured correctly.
 /// It is intended to be used by the L1 PAO multisig to validate the configuration of the L1 contracts
 /// before and after an upgrade.
-contract OPContractsManagerStandardValidator is ISemver {
-    /// @notice The semantic version of the OPContractsManagerStandardValidator contract.
-    /// @custom:semver 1.5.0
-    string public constant version = "1.5.0";
+contract StandardValidator is ISemver {
+    /// @notice The semantic version of the StandardValidator contract.
+    /// @custom:semver 1.1.2
+    string public constant version = "1.1.2";
```

Comment:
Why did this get renamed and the version decremented? I think it's the cause of some of the test failures with op-deployer.